welder-web.spec

```diff
@@ -0,0 +1,25 @@
+Name: welder-web
+Version: 0
+Release: 0
```

Comment:
Why not use Tito to take care of this, since you probably want to publish changelogs and tag releases as well.
AFAIK RPM doesn't allow you to set the Version tag dynamically so the value has to be changed externally.